vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -82,7 +78,8 @@
     <Import Project="PropertySheet.props" />
     <Import Project="..\PropertySheets\WinUI.props" />
   </ImportGroup>
-  <Import Project="$(SolutionDir)packages\$(WinUIPackageProps)" Condition="'$(WinUIPackageProps)'!='' And Exists('$(SolutionDir)packages\$(WinUIPackageProps)')" />
+  <!-- TODO: DELETE? -->
+  <!-- <Import Project="$(SolutionDir)packages\$(WinUIPackageProps)" Condition="'$(WinUIPackageProps)'!='' And Exists('$(SolutionDir)packages\$(WinUIPackageProps)')" /> -->
```

Comment:
Does this impact functionality?

Reply:
we still need to consume the right WinUI package, either winui2.x or winui 3

Reply:
It may. Right now, I am unable to compile that project from the command line and it seems related to WinUI. I'll consult with @asklar to fix it.

The rest of the projects restore and build fine.

Note, this is work in progress as I still need to address CI changes and potentially doc updates.

Reply:
Done.